ble_controller/README.rst

```diff
@@ -1,17 +1,17 @@
 .. _ble_controller:
 
 nRF Bluetooth LE Controller
-===========================
+###########################
 
-| The nRF Bluetooth LE Controller is an RTOS-agnostic library built for the Nordic Semiconductor nRF52 and nRF53 Series that supports Bluetooth 5.
-| In addition to providing a BLE HCI interface, it also provides:
+The |controller| is an RTOS-agnostic library built for the Nordic Semiconductor nRF52 and nRF53 Series that supports Bluetooth 5.
+In addition to providing a Bluetooth LE HCI interface, it also provides:
```

Comment:
The Bluetooth terminologi is just `Bluetooth HCI interface`. I would therefore prefer to keep it as is.

Reply:
We should change to Bluetooth HCI interface then, not leave as BLE HCI interface?